docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,206 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations including procedures for customizing and applying any
+given Cluster and Services Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
+
+Newly deployed **MetalK8s** clusters come with chosen default values for most
+Cluster services. These default values are transparent to Admin users
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
+
+Managing Authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Add a local static user
+"""""""""""""""""""""""
+
+Local authentication via static users is enabled by default after a fresh
+MetalK8s installation.
+
+   .. important::
+
+      To continue using MetalK8s in cases where the external authentication
+      system fails, we advise MetalK8s administrators to leave the default
+      super admin account enabled at all times.
+
+To add a new static user, perform the following operations:
+
+#. Generate a password hash. The command below requires **apache-utils**
+   package to be installed.
+
+   .. code-block:: shell
+
+      $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+
+#. From the Bootstrap node, edit the ConfigMap ``metalk8s-dex-config`` and then
+   add a new entry to the already existing list using:
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+         edit configmaps metalk8s-dex-config -n metalk8s-auth
+
+   The new entry should be unique and possess mandatory fields like ``email``,
+   ``hash``, ``username`` and ``userID`` like in the example below.
+
+   .. code-block:: yaml
+
+      data:
+         config.yaml: |-
+            spec:
+               localuserstore:
+                  userlist:
+                    - email: "<email>"
+                      hash: "<replace-with-hash>"
+                      username: "<username>"
+                      userID: "<uuidv4>"
+
+#. Save and apply the changes.
+
+
+#. Connect to the Salt master container using:
```

Comment:
Couldn't we merge the 2 following actions in one single command line?

Reply:
We shouldn't

Reply:
why?

Reply:
I was thinking it is easier to copy/paste the two individual commands than having one very long command from the docs.

But as you want, we can `&&` the two commands making them one

Reply:
@xaf-scality @bkettler what do you think?

Reply:
If you do in two separate command for me you need to say that you need to leave the container bash before executing the kubectl one.

For me since you only have one command to run in the container having a single command looks better IMO (but I let Xavier and Billy answer for this one as they are more concerned than me)

Reply:
Ok, I do not mind combining them to =>

`crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash -c "salt-run state.sls metalk8s.addons.dex.deployed metalk8s-<version>"`

Reply:
fixed!